src/libsystemd/sd-device/device-enumerator.c

```diff
@@ -118,7 +118,7 @@ _public_ int sd_device_enumerator_add_match_sysattr(sd_device_enumerator *enumer
         else
                 hashmap = &enumerator->nomatch_sysattr;
 
-        r = hashmap_put_strdup(hashmap, sysattr, value);
+        r = hashmap_put_strdup_full(hashmap, false, sysattr, value);
```

Comment:
I don't get this: what is so special about this call site that we don't want to hash the strings?

Reply:
Before the offending commit, multiple sysattr or property values can be specified as we used `trivial_hash_ops`. With `string_hash_ops` we cannot set multiple values for the same key.

Now the commit message also mentions that.